CHANGELOG.rst

```diff
@@ -1,3 +1,11 @@
+0.3.1 (2021-11-18)
+==================
+
+Bug Fixes
+---------
+
+- Fixed issue with ``time_lag`` and dask arrays.
```

Comment:
What is the issue?

Reply:
See https://dev.azure.com/sunpy/sunkit-image/_build/results?buildId=21411&view=logs&jobId=0609049e-3a2a-5217-995e-2cc283dcaa52&j=0609049e-3a2a-5217-995e-2cc283dcaa52&t=8711ded8-03e2-5951-4fd3-69b4a92338f7